src/yb/tools/yb-admin_cli.cc

```diff
@@ -83,6 +83,17 @@ CHECKED_STATUS ChangeBlacklist(ClusterAdminClientClass* client,
   return Status::OK();
 }
 
+CHECKED_STATUS MasterLeaderStepDown(
+    ClusterAdminClientClass* client,
+    const ClusterAdminCli::CLIArguments& args){
+  if(args.size()<3){
+    return ClusterAdminCli::kInvalidArguments;
+  }
+  RETURN_NOT_OK_PREPEND(client->MasterLeaderStepDownWithNewLeader(
+          args[2]), "Unable to step down master leader");
+  return Status::OK();
```

Comment:
`return client->MasterLeaderStepDownWithNewLeader(args[2]);`

Reply:
BTW it is not really necessary to create new client's method `MasterLeaderStepDownWithNewLeader`. It is better to find current leader in current lambda and use existed `MasterLeaderStepDown` method.